GliaWidgets/Sources/CallVisualizer/CallVisualizer.swift

```diff
@@ -131,31 +134,41 @@ extension CallVisualizer {
 
 extension CallVisualizer {
     func startObservingInteractorEvents() {
-        environment.interactorProviding()?.addObserver(self) { [weak self] event in
+        interactorSubscription = environment.interactorPublisher
```

Comment:
Could you add a test checking that `CallVisualizer.activeInteractor` is updated once Glia.sharedInstance.interactor` is re-created?

Reply:
fixed [00a5146](https://github.com/salemove/ios-sdk-widgets/pull/1198/commits/00a51464b7b980c0480ffd18093c4f2cf2a682f6)